src/environmentd/src/http/static/js/memory.jsx

```diff
@@ -27,52 +27,148 @@ async function query(sql) {
 
 const { useState, useEffect } = React;
 
-function useSQL(sql) {
-  const [response, setResponse] = useState(null);
+function ReplicaView() {
+  const [currentClusterName, setCurrentClusterName] = useState(null);
+  const [currentReplicaName, setCurrentReplicaName] = useState(null);
+  const [sqlResponse, setSqlResponse] = useState(null);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(false);
 
+  const queryClusterReplicas = `
+    SELECT
+    clusters.name AS cluster_name, replicas.name AS replica_name
+    FROM
+      mz_catalog.mz_cluster_replicas replicas
+      LEFT JOIN mz_catalog.mz_clusters clusters ON clusters.id = replicas.cluster_id
+    ORDER BY replicas.id asc
```

Comment:
```suggestion
    ORDER BY cluster_name, replicas.name
```

Might be easier for folks to visually find what they want.

Reply:
I used id as a proxy for order of creation, and thus guaranteeing the default cluster shows up first (because we'll automatically load data for the default/first selection), but happy to do any ordering.

Reply:
Good point. This could be addressed by setting `defaultValue` below to `default` (if that cluster exists, otherwise...no default?).

Reply:
Yeah I was hesitant to hardcode the word `default` in case we change that down the road, but I think it's fine.